frontend/src/components/Displays/MissionDisplays/MissionStatusDisplay.tsx

```diff
@@ -21,7 +21,7 @@ const StyledStatusIcon = styled.div`
     align-items: flex-end;
 `
 
-export function displayIcon(status: MissionStatus) {
+export const DisplayMissionStatusIcon = ({ status }: StatusProps) => {
```

Comment:
Could we just call it MissionStatusIcon? Since the "display" part is kind of a given